frontend/src/components/editor/Editor.tsx

```diff
@@ -73,18 +73,18 @@ function Editor() {
 					checkAndAddFormat("`", FormatType.CODE);
 					checkAndAddFormat("~~", FormatType.STRIKETHROUGH);
 
-					setFormatBarState((prev) => ({
+					setToolBarState((prev) => ({
 						...prev,
 						show: true,
 						position: {
 							top: coords.top - 8,
-							left: coords.left + 190,
+							left: coords.left + 30,
 						},
 						selectedFormats: formats,
 					}));
```

Comment:
It seems that we should change the position when changing the `FormatBar` component. (Is it right?)
For now, it is okay because this change will resolve the https://github.com/yorkie-team/codepair/issues/272, https://github.com/yorkie-team/codepair/issues/205.

Could you add the comment for this `TODO`?
```
TODO: Change the FormatBar rendering method.
```